frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/providers/v2vvmware-tab-validation.ts

```diff
@@ -18,14 +18,25 @@ export const getV2VVMwareImportProvidersTabValidity = (
 
   const hasSelectedVM = !!iGetVMWareFieldValue(state, id, VMWareProviderField.VM);
   const hasLoadedVM = !!iGetVMWareFieldAttribute(state, id, VMWareProviderField.VM, 'vm');
+  // check if host name has any kind of whitespace
```

Comment:
this logic should be in a new function called validateV2VVMwareImportProvidersTab. 
We could also keep the in validationConfig format as a convention (you can take an inspiration from `ovirt-provider-tab-validation`)

Reply:
also the check and save button should reflect the error state of this validation